salt/modules/win_pkg.py

```diff
@@ -692,10 +791,15 @@ def genrepo(**kwargs):
                     ret,
                     successful_verbose
                     )
-    with salt.utils.fopen(repo_details.winrepo_file, 'w+b') as repo_cache:
-        repo_cache.write(msgpack.dumps(ret))
+    serial = salt.payload.Serial(__opts__)
+    # TODO: 2016.11 has PY2 mode as 'w+b' develop has 'w+' ? PY3 is 'wb+'
+    # also the reading of this is 'rb' in get_repo_data()
+    mode = 'w+' if six.PY2 else 'wb+'
+    # with salt.utils.files.fopen(repo_details.winrepo_file, mode) as repo_cache:
```

Comment:
This doesn't need to be here, it'll be caught once this would merge forward to develop and it's a simple matter of adding the `.files` to the function call.